src/ucp/wireup/wireup_cm.c

```diff
@@ -17,6 +17,24 @@
 #include <ucs/sys/string.h>
 
 
+/* if CM callback is invoked for CM lane of UCP EP which is marked as
+ * failed one, i.e. the failing UCP is in progress, because UCT CM EP is
+ * still alive (e.g. err_handling progress part was scheduled on worker
+ * or discarding) - so, don't do any actions in the callback -
+ * discarding functionality will destroy UCT EP */
+#define UCP_CM_CB_CHECK_UCP_EP(_ucp_ep, _uct_cm_ep, _exit_action) \
```

Comment:
1. UCP_EP_CM_CALLBACK_ENTER(_ucp_ep, _uct_cm_ep, _failed_action)
2. The doc is unclear/confusing. Pls replace by just a short doc of macro args